<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSH Login</string>
  </property>
  <widget class="QLineEdit" name="lneUsername">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>171</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnePassword">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>171</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>92</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>92</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>clickedCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>clickedOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
